flask_tutorial-code_review/main.py

```diff
@@ -0,0 +1,71 @@
+from flask import Flask, render_template, request, redirect, url_for, session, flash
+from flask_pymongo import PyMongo
+from datetime import datetime
+from flask_migrate import Migrate
+from config import DevelopmentConfig
+
+app = Flask(__name__, static_url_path='/static')
+app.config['MONGO_URI'] = 'mongodb://localhost:27017/yourdatabase'
```

Comment:
you provide mongodb url? why not use the one you created in config.py

Reply:
done